src/client/driver/command-executors/execute-action.js

```diff
@@ -116,9 +117,19 @@ function ensureCommandArguments (command) {
     }
 }
 
+function ensureCommandOffsetOptions (element, command) {
+    var { offsetX, offsetY } = getOffsetOptions(element, command.options.offsetX, command.options.offsetY);
+
+    command.options.offsetX = offsetX;
+    command.options.offsetY = offsetY;
+}
+
 function createAutomation (elements, command) {
     var selectArgs = null;
 
+    if (command.options && 'offsetX' in command.options && 'offsetY' in command.options)
```

Comment:
Unfortunately this doensn't work. Object `command` comes via xhr, so it's just key\value object, not instance of class.